test/Driver/loaded_module_trace_env.swift

```diff
@@ -6,7 +6,18 @@
 // CHECK: "name":"loaded_module_trace_env"
 // CHECK: "arch":"{{[^"]*}}"
 // CHECK: "swiftmodules":[
-// CHECK: "{{[^"]*\\[/\\]}}Swift.swiftmodule{{(\\[/\\][^"]+[.]swiftmodule)?}}"
-// CHECK: "{{[^"]*\\[/\\]}}SwiftOnoneSupport.swiftmodule{{(\\[/\\][^"]+[.]swiftmodule)?}}"
+// CHECK-DAG: "{{[^"]*\\[/\\]}}Swift.swiftmodule{{(\\[/\\][^"]+[.]swiftmodule)?}}"
+// CHECK-DAG: "{{[^"]*\\[/\\]}}SwiftOnoneSupport.swiftmodule{{(\\[/\\][^"]+[.]swiftmodule)?}}"
```

Comment:
This is a bit concerning. Is it not deterministic?

Reply:
It is deterministic. The point (at least in my head) is that the test doesn't care about the order itself, it only cares that these elements are present.

Reply:
Hm, okay.